.github/workflows/startRelease.yml

```
@@ -50,6 +50,8 @@ jobs:
       - name: push branch
         run: |
           git checkout -b release-candidate
+          yarn export:ui-props
+          git add public/exports/smarthr-ui-props.json && git diff --cached --exit-code --quiet || git commit -m 'Update smarthr-ui-props.json'
```

GitHub ActionsにJSON作成スクリプトの実行と、JSONに差分が出ればコミットするgitコマンドを追加しました。
この場所で問題ないかなどレビューいただければと思います。

---

コミットメッセージは conventional commits のフォーマットになっている必要があるので、以下のように修正した方が良いかなと思いました！

```suggestion
          yarn export:ui-props
          git add public/exports/smarthr-ui-props.json && git diff --cached --exit-code --quiet || git commit -m 'chore: Update smarthr-ui-props.json'
```